lib/utils/gasEstimation.js

```diff
@@ -185,7 +170,7 @@ const stepTracker = () => {
       allOps.push(info);
     },
     isSVT: (index) => svt.includes(index),
-    success: () => !allOps.length || isTerminator(allOps[allOps.length - 1].opcode.name),
+    success: () => !allOps.length || !isTerminator(allOps[allOps.length - 1].opcode.name),
```

Comment:
This feels backwards to me. I feel like the result should be negated here (as well as the `steps.success` check above)